src/streaming/models/CmcdModel.js

```diff
@@ -216,6 +268,11 @@ function CmcdModel() {
             data.pr = pr;
         }
 
+        // Assuming nor field is never set: If the ‘nor’ field is not set, then the object is assumed to match the object currently being requested.
+        if (request.range) {
+            data.nrr = request.range;
```

Comment:
See above. This refers to omitting the nor object in case nnr is present and the nor object did not change.